wifiphisher/common/phishingpage.py

```diff
@@ -317,6 +317,33 @@ def get_templates(self):
 
         return self._templates
 
+    def search_template(self, name):
+        """
+        Check the directory for config and html
+        :param self: A TemplateManager object
+        :param name: A directory name
+        :type self: A TemplateManager object
+        :return: True or False
+        :rtype: None
+        """
+        config = False
+        html = False
+        # directory files
+        dir_files = os.listdir(os.path.join(self._template_directory, name))
+        # check all files in directory on...
+        for files in dir_files:
+                    # html
+            if files.find(".html", 0) != -1:
+                html = True
+            # config
+            elif files.find("config.ini", 0) != -1:
+                config = True
+            #and if we found them all return true
```

Comment:
remove four indentations here (i.e. The comment should start with the same column as the following statement) and left space after `#`. (i.e. `# and if we ...`)